src/applications/hca/actions.js

```diff
@@ -8,6 +8,7 @@ import {
 } from './selectors';
 
 // flip the `false` to `true` to fake the endpoint when testing locally
+// eslint-disable-next-line sonarjs/no-redundant-boolean
```

Comment:
ESLint was disabled since the following line is using a false/true as a toggle for testing purposes.

Reply:
😬